Modules/Registration/Metricsv4/test/itkObjectToObjectMultiMetricv4Test.cxx

```diff
@@ -44,6 +44,10 @@ using ObjectToObjectMultiMetricv4TestMultiMetricType = itk::ObjectToObjectMultiM
 
 int itkObjectToObjectMultiMetricv4TestEvaluate( ObjectToObjectMultiMetricv4TestMultiMetricType::Pointer & multiVariateMetric, bool useDisplacementTransform )
 {
+  //The summations in the derivative depend on the ordering.
+  //Multithreading can introduce noticably different results
+  itk::MultiThreaderBase::New()->SetMaximumNumberOfThreads(1);
```

Comment:
Perhaps: `itk::MultiThreaderBase::SetGlobalMaximumNumberOfThreads(1);`?

Reply:
I removed this change again. I added it due to the test initially failing which was fixed when the fix https://github.com/InsightSoftwareConsortium/ITK/pull/1037 was incorporated into this branch (the fix in 1037 removes the issue of threads writing/reading to the same temporary variable which messed up the results.)